packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/inspections/impl/FieldCheckInspectionBridge.kt

```diff
@@ -177,7 +177,7 @@ internal object FieldCheckLinterInspection : MongoDbInspection {
             descriptor: ProblemDescriptor,
         ) {
             val editor = FileEditorManager.getInstance(project).selectedTextEditorWithRemotes[0] ?: return
-            MdbJavaEditorToolbar.showModalForSelection(editor, coroutineScope)
+            // MdbJavaEditorToolbar.showModalForSelection(editor, coroutineScope)
```

Comment:
I'm not sure this should be commented?

```suggestion
            MdbJavaEditorToolbar.showModalForSelection(editor, coroutineScope)
```